doc/ontodoc/README.md

```diff
@@ -0,0 +1,33 @@
+Generate documentation for BattINFO
+===================================
+This directory contains the needed templates, introductory text and
+figures for generating a reference documentation of BattINFO.
+
+The documentation is generated using EMMO-python and pandoc. Just run the
+script
```

Comment:
```suggestion
This directory contains the needed templates, introductory text and figures for generating a reference documentation of BattINFO.

The documentation is generated using EMMO-python and pandoc.
Just run the script
```
This is something I learned from collaborating on the AiiDA documentation.
It's much easier to decipher markdown and rich-text git diffs if you have one sentence per one line. If you split sentences over several lines and change its structure, git diffs can sometimes come out looking very confusing, even though the change may be small. But if it's all in the same line it usually works better and is usually much clearer. However that said, git diffs for non-code documents can always be subject to confusion, simply because it's not the syntax git diffs are developed for, I presume?

Another contention here, might be your editor that isn't optimal at wrapping long lines, in which case I would prefer the editor showing something meaningful than the git diffs being nice.

Reply:
Thank you for the tip. It is not difficult to make emacs word-wrap long lines instead of wrapping at 80 characters...

Reply:
Okay. So should I try and go through everything and apply this "one sentence per one line" thing?